flink-end-to-end-tests/test-scripts/test_kubernetes_itcases.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+source "$(dirname "$0")"/common_kubernetes.sh
+
+start_kubernetes
+
+# Set the ITCASE_KUBECONFIG environment since it is required to run the ITCases
+export ITCASE_KUBECONFIG=~/.kube/config
+
+cd $END_TO_END_DIR/../flink-kubernetes
```

Comment:
Do you mean we start the minikube in the `KubernetesResource`? Or we assume that we have an prepared minikube. Currently, we are using this script to start the minikube and run tests.

If you already have a minikube running on your machine, set the `ITCASE_KUBECONFIG` environment, then we could directly run the test in IDE or maven.

Reply:
I meant to start minikube in the `MinikubeResource` or something similar. But this would have the disadvantage that we could not reuse a minikube cluster/instance for different IT cases.

Reply:
Yes. It will make the ITCase run longer as the Kubernetes ITCases get more and more. Currently, I will try to keep the script here. Actually, if you want to debug the ITCase in your IDE or run in your local machine with the `test_kubernetes_itcases.sh`. Both of them could work.